recipe/build.sh

```diff
@@ -0,0 +1,29 @@
+
+rm -rf build
+
+# ModuleNotFoundError: No module named '_sysconfigdata_x86_64_apple_darwin20_0_0'
+unset _PYTHON_SYSCONFIGDATA_NAME
+unset _CONDA_PYTHON_SYSCONFIGDATA_NAME
+
```

Comment:
That looks wrong. We need to understand what's going on.

Reply:
The switch to ninja causes the selection of the wrong `_CONDA_PYTHON_SYSCONFIGDATA_NAME`:

```
                   INFO   +_CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_apple_darwin20_0_0                                                                                                         
```

with a subsequent:
```
                   INFO   ModuleNotFoundError: No module named '_sysconfigdata_x86_64_apple_darwin20_0_0'                                                                                                    
```

https://cloud.prefect.io/anaconda/flow-run/fc5c0bf3-7f37-4524-a694-ddef3cdc08a9

While before, when it was building:

```
                   INFO   +_CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_apple_darwin13_4_0                                                                                                         
```
https://cloud.prefect.io/anaconda/flow-run/ec7232ca-3064-4fcc-be22-fc20a00f9d04

my next attempt would be a `recipe/conda_build_config.yaml` with something like:
```
FINAL_PYTHON_SYSCONFIGDATA_NAME:
  - _sysconfigdata_x86_64_apple_darwin13_4_0  # [x86_64]
  - _sysconfigdata_arm64_apple_darwin20_0_0   # [not x86_64]
```
as in `clang-compiler-activation-feedstock/recipe/conda_build_config.yaml`?

Reply:
The problem is not the switch to ninja. The problem is a new build of `lang-compiler-activation-feedstock` that was pushed last night. Actually, the problem is not the rebuild itself and it's more that we'll need to fix python.

Reply:
ah ok, I saw https://github.com/AnacondaRecipes/aggregate/pull/316/files I will wait.